docs/source/guide.rst

```diff
@@ -277,15 +277,14 @@ For example this definition of Tempest object:
 will ensure that tempest will be executed with tempest.conf that looks like this:
 
 
-.. code-block:: toml
```

Comment:
I do not know why but the job started to fail. It was complaining about the toml formatting.